### ISSUE TITLE
Itinerary Details - Enhancement - Display Start/End Address on Map

### ISSUE THREAD

**Issue by stevejgordon** [HTBox/allReady#1572]
Following on from #1571 we should be able to display the Start/End address on the itinerary details map (when an address is set).

**Comment by ExoltenOne** [comment]
@stevejgordon , is this one ready to go? If yes, could I take it?

**Comment by stevejgordon** [comment]
Yeah this is up for grabs. I've got a small job planned in the same files but I can finish that after.

**Comment by ExoltenOne** [comment]
Cool, so I'm taking it.

**Comment by stevejgordon** [comment]
Hi @ExoltenOne - Just checking in to see if you were still working on this / have something you can submit?
